.github/workflows/build.yml

```diff
@@ -56,14 +58,50 @@ jobs:
       - name: Test
         run: dotnet test OpenUtau.Test
 
-      # Build
+      # Build: Main
       - name: Restore
         run: dotnet restore OpenUtau -r ${{ matrix.arch.rid }}
 
       - name: Publish
         run: dotnet publish OpenUtau -c Release -r ${{ matrix.arch.rid }} --self-contained true -o bin/${{ matrix.arch.name }}/
         if: ${{ matrix.arch.os != 'osx' }}
 
+      # Build: Daw Plugin
+
+      - name: Build Daw Plugin (Windows)
+        run: |
+          cd ./DawPlugin
+          cmake -B build -S . -DCMAKE_BUILD_TYPE=Release
+          cmake --build build --config Release
+        if: ${{ matrix.daw-plugin && matrix.arch.os == 'win' }}
+
+      - name: Build Daw Plugin (Mac)
+        run: |
+          cd ./DawPlugin
+          sudo xcode-select -s /Applications/Xcode_15.4.app/Contents/Developer
+          cmake -B build -S . -DCMAKE_BUILD_TYPE=Release
+          cmake --build build --config Release
+        if: ${{ matrix.daw-plugin && matrix.arch.os == 'osx' }}
```

Comment:
- Build on MacOS and AU format: 

I don't have MacOS device.
I tried to build on GH Actions, but it was difficult (and frustrating) for me...